data/json/items/tool/misc.json

```
@@ -58,6 +83,70 @@
     "color": "brown",
     "use_action": "BELL"
   },
+  {
+    "type": "TOOL",
+    "id": "eac_dab_pen",
+    "name": "dab pen",
```

```suggestion
    "name": { "str": "dab pen" },
```
When the plural is just adding an `s` at the end you don't need to define it, the game handles it